Simplenote/src/main/java/com/automattic/simplenote/widgets/NoteEditorViewPager.java

```diff
@@ -18,6 +19,7 @@ public boolean onInterceptTouchEvent(MotionEvent event) {
         return this.mIsEnabled && super.onInterceptTouchEvent(event);
     }
 
+    @SuppressLint("ClickableViewAccessibility")
```

Comment:
```suggestion
    @Override
    public boolean onTouchEvent(MotionEvent event) {
        performClick();
        return this.mIsEnabled && super.onTouchEvent(event);
    }

    @Override
    public boolean performClick() {
        return this.mIsEnabled && super.performClick();
    }
```

Rather than suppressing this warning, we may want to override the `performClick()` method mentioned in the warning.  I haven't tested or researched the consequences of this, but ignoring the warning could be an anti-pattern.

Reply:
Hey @theck13! I tried to do some research about this and I think we should call `performClick()` only on `MotionEvent.ACTION_UP` action but I'm not sure my implementation is the right one.

Please let me know if you prefer that I use this approach instead or have any other suggestion.

Reply:
Good idea about the `MotionEvent.ACTION_UP` action.  I think your solution works.  We might want to update the `onTouchEvent` method like the code below.  It's effectively the same thing as your solution, but it uses the early exit pattern used throughout the codebase.  What do you think?

```java
    @Override
    public boolean onTouchEvent(MotionEvent event) {
        if (!mIsEnabled) {
            return false;
        }

        if (event.getAction() == MotionEvent.ACTION_UP) {
            performClick();
        }

        return super.onTouchEvent(event);
    }
```